scripts/kafka-integration-test.sh

```diff
@@ -1,44 +1,94 @@
 #!/bin/bash
 
-set -e
+set -euxf -o pipefail
 
-export STORAGE=kafka
-compose_file="docker-compose/kafka-integration-test/docker-compose.yml"
+usage() {
+  echo $"Usage: $0 -k <test_version>"
```

Comment:
help should explain what parameters mean
```suggestion
  echo $"Usage: $0 -k <jaeger_version>"
```